assets/css/default.css

```diff
@@ -9,6 +9,33 @@ body {
     Arial, sans-serif;
 }
 
+.length-watch-page{
+  position: relative;
+  background-color: rgba(35, 35, 35, 0.75);
+  color: #fff;
+  border-radius: 2px;
+  padding: 4px;
+  font-size: 20px;
+  right: 0.25em;
+  bottom: 0.25em;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+
+.dark-theme .length-watch-page{
+  position: relative;
+  background-color: #fff;
+  color: rgba(35, 35, 35, 1);
```

Comment:
You're going to have to specify these again under the `@media (prefers-color-scheme: dark)` media query unfortunately as to support automatic theming